libraries/SPI/SPI.h

```diff
@@ -169,13 +170,18 @@ class SPIClass {
   char interruptSave;
   uint32_t interruptMask;
 
-  // transfer(txbuf, rxbuf, count, block) uses DMA if possible
-  Adafruit_ZeroDMA readChannel,
-                   writeChannel;
-  DmacDescriptor  *readDescriptor  = NULL,
-                  *writeDescriptor = NULL;
-  volatile bool    dma_busy = false;
-  static void      dmaCallback(Adafruit_ZeroDMA *dma);
+  // objects and functions used for dma transfer
+  Adafruit_ZeroDMA readChannel;
+  Adafruit_ZeroDMA writeChannel;
+  DmacDescriptor  *readDescriptor  = NULL;
+  DmacDescriptor  *writeDescriptor = NULL;
```

Comment:
given this doesn't change functionality and the original was correct, probably should not be changed.

Reply:
Yeah this is a stye choice, i'm usually explicit with variable names one every line